recipes-core/images/onie-nos-installer.inc

```diff
@@ -16,7 +16,7 @@ BISDN_ONIE_PLATFORM ?= "${@'${ONIE_PLATFORM}'.replace('_','-')}"
 
```

Comment:
In the commit message for "installer: flatten machines and use ONIE names",  s/a a /a /.

Also, we should document somewhere when it is _not_ sufficient to use a symlink for two similar platforms.

Reply:
Fixed, and reworded a bit to make it more clear why a symlink is sufficient here.